enzyme/CMakeLists.txt

```diff
@@ -210,8 +211,10 @@ export(TARGETS ClangEnzyme-${LLVM_VERSION_MAJOR}
     APPEND FILE "${PROJECT_BINARY_DIR}/EnzymeTargets.cmake")
 endif()
 
-export(TARGETS LLDEnzyme-${LLVM_VERSION_MAJOR}
-    APPEND FILE "${PROJECT_BINARY_DIR}/EnzymeTargets.cmake")
+if (NOT ${ENZYME_BUILD_STATIC_ONLY})
```

Comment:
Rather than having a build static only flag, can you instead have a BUILD_ENZYMELLD flag, which is default to on here?